Code/Framework/AzToolsFramework/AzToolsFramework/Component/EditorComponentAPIComponent.cpp

```diff
@@ -597,11 +597,16 @@ namespace AzToolsFramework
                 pte.SetVisibleEnforcement(true);
             }
 
+            AzToolsFramework::UndoSystem::URSequencePoint* modifyPropertyUndoOperation;
+            ToolsApplicationRequests::Bus::BroadcastResult(
+                modifyPropertyUndoOperation, &ToolsApplicationRequests::BeginUndoBatch, "Modify Entity Property");
+            ToolsApplicationRequests::Bus::Broadcast(&ToolsApplicationRequests::AddDirtyEntity, componentInstance.GetEntityId());
```

Comment:
You should be able to replace this with a ScopedUndoBatch since it does the same operations

Reply:
updated and tested, thanks